### ISSUE TITLE
Invalid integer value: 9288.0 exception

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#315]
```
Hi,
I'm getting 
org.apache.poi.xwpf.converter.core.XWPFConverterException: 
org.apache.xmlbeans.impl.values.XmlValueOutOfRangeException: Invalid integer 
value: 9288.0

while I'm trying convert my docx file to pdf using 
org.apache.poi.xwpf.converter.pdf, ver. 1.0.2

My code:
InputStream in = new FileInputStream(new  
File("/home/dstachyra/test/Test1.docx"));
XWPFDocument document = new XWPFDocument(in);

// 2) Prepare Pdf options
PdfOptions options = PdfOptions.create();

// 3) Convert XWPFDocument to Pdf
OutputStream out = new FileOutputStream(new 
File("/home/dstachyra/test/Test1.pdf"));

PdfConverter.getInstance().convert(document, out, options);

I'm using Ubuntu 13.10, Java 1.6.

File used to test attached.


```

Original issue reported on code.google.com by `dariusz....@gmail.com` on 23 Oct 2013 at 7:16

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Could you attach your docx please.

Many thank's.

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 23 Oct 2013 at 8:23
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dariusz....@gmail.com` on 23 Oct 2013 at 8:40

Attachments:
- [Test1.docx](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-315/comment-2/Test1.docx)

**Comment by GoogleCodeExporter** [comment]
```
Pb is fixed for 1.0.3.

See Git commit at 
https://code.google.com/p/xdocreport/source/detail?r=dd1b13a80997d950ba37f0c7119
2382e36d60eaa

You can test it with our online demo at 
http://xdocreport-converter.opensagres.cloudbees.net/
```

Original comment by `angelo.z...@gmail.com` on 23 Oct 2013 at 11:46
- Changed state: **Fixed**
